sympy/polys/ring_series.py

```diff
@@ -1638,7 +1642,7 @@ def rs_tanh(p, x, prec):
                 const = R(tanh(c))
             except ValueError:
                 raise DomainError("The given series can't be expanded in this "
-                "domain")
+                    "domain")
```

Comment:
What should you align it to?

Sent from my mobile phone.
On Jul 10, 2015 12:01 PM, "Sartaj Singh" <notifications@github.com> wrote:

> In sympy/polys/ring_series.py
> <https://github.com/sympy/sympy/pull/9640#discussion_r34374081>:
>
> > @@ -1638,7 +1642,7 @@ def rs_tanh(p, x, prec):
> >                  const = R(tanh(c))
> >              except ValueError:
> >                  raise DomainError("The given series can't be expanded in this "
> > -                "domain")
> > +                    "domain")
>
> Output by flake8.
>
> sympy/polys/ring_series.py:1645:21: E128 continuation line under-indented
> for visual indent
>
> By aligning ", this can be fixed.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/sympy/sympy/pull/9640/files#r34374081>.
>